test/standalone/child_process_setup/build.zig

```diff
@@ -0,0 +1,20 @@
+const std = @import("std");
+
+pub fn build(b: *std.build.Builder) void {
+    const target = b.standardTargetOptions(.{});
+    const mode = b.standardReleaseOptions();
+    const exe = b.addExecutable("child_process_setup", "main.zig");
+    exe.setTarget(target);
```

Comment:
Do you expect miscompilations on Release? Otherwise, Debug mode should be sufficient/no setting of target and buildmode. See #11736 (has unfortunately weird CI errors).

Reply:
Looks like there's no reason for the build mode, but I'm currently using the target to check if it's Darwin or not to disable the test in that case.